apps/organisations/forms.py

```diff
@@ -136,7 +135,7 @@ class OrganisationForm(forms.ModelForm):
         ),
         (
             "information",
-            RichTextUploadingFormField,
+            CKEditor5Widget,
             {
                 "config_name": "collapsible-image-editor",
                 "label": OrganisationTranslation._meta.get_field(
```

Comment:
@goapunk label gives error
```
TypeError: __init__() got an unexpected keyword argument 'label'
```
I guess label is not part of CKEditor5Widget?

Reply:
It's sadly not inheriting from `CharField` I think. We need to set the label in `__init__` I think